### DISCUSSION TITLE
Misdirected accounts are flagged and have not responded for more than 1 month.

### DISCUSSION THREAD

**Discussion by afirmansyah44** [community/community#116305] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

![ce4a74d6-755b-42c8-946d-6ae22e921929](https://github.com/community/community/assets/147068684/1c96f5bd-3798-46c1-8e91-746cc8a57efa)

Hello everyone, I hope this message can be read by the GitHub team or interested parties. I have a colleague with a case of a "flagged" GitHub account. He has reported it via GitHub support since 02/21/2023 and as of now 03/30/2023 he has not received any response. The incident started when my friend's GitHub username was tagged in someone else's GitHub repository issue but the message contained a FRAUD message which contained an invitation to work remotely with a high salary and was directed to a link included by the fraudster, which in essence would hack his GitHub account if he clicked on it. and you repeatedly received it so it's possible that github was wrong and thought you were a fraud team. this is my friend's username: @damartripamungkas 

below is a snippet of the fraudulent message.

**Comment by Mays-M** [comment]
interesting

**Comment by LiteBrite82** [comment]
Hi @afirmansyah44!
The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the General category. I've gone ahead and moved it for you. Good luck!

**Comment by gokhan-or** [comment]
> ### Select Topic Area
> 
> Question
> 
> ### Body
> 
> 
> ![ce4a74d6-755b-42c8-946d-6ae22e921929](https://github.com/community/community/assets/147068684/1c96f5bd-3798-46c1-8e91-746cc8a57efa)
> 
> Hello everyone, I hope this message can be read by the GitHub team or interested parties. I have a colleague with a case of a "flagged" GitHub account. He has reported it via GitHub support since 02/21/2023 and as of now 03/30/2023 he has not received any response. The incident started when my friend's GitHub username was tagged in someone else's GitHub repository issue but the message contained a FRAUD message which contained an invitation to work remotely with a high salary and was directed to a link included by the fraudster, which in essence would hack his GitHub account if he clicked on it. and you repeatedly received it so it's possible that github was wrong and thought you were a fraud team. this is my friend's username: @damartripamungkas 
> 
> below is a snippet of the fraudulent message.
